app/views/layouts/_navigation.html.erb

```diff
@@ -1,48 +1,65 @@
-<nav class = "navbar navbar-expand-lg bg-body-tertiary">
-  <div class = "container-fluid">
-    <a class = "navbar-brand" href="<%= root_path %>">Cinema</a>
-    <button class = "navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Try to not use such long lines as for it's difficult to read in an editor, you always have to move the horizontal scrollbar back and forth to read code